skeleton/github/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+https://github.com/nimblehq/git-template/issues/??
```

Comment:
Should we use the `Close` keyword here? :D 

```suggestion
- Close #
```

If you agree, please approve this PR: https://github.com/nimblehq/git-template/pull/32 😇

Reply:
I agree. But you should mark that PR ready for review 😄

Reply:
Added in https://github.com/nimblehq/infrastructure-templates/pull/97/commits/2963dbf060c1f027293ed8f05ef9b51d45d512a9